students/opcode6502/lesson_05/mailroom_03.py

```diff
@@ -0,0 +1,354 @@
+# mailroom_03.py
+# opcode6502: SP_Online_PY210
+
+
+import datetime
+import sys
+import tempfile
+
+
+def add_donation(donor_name, donation_amount):
+    try:
+        donors_db[donor_name] = float(donation_amount)
+        print_debug_message('add_donation(donor_name, donation_amount): Success!')
+        return True
+    except:
```

Comment:
All of these the exceptions in this code are 'bare' except statements.  You want to try and be as explicit as possible.  Catch specific types of errors as much as possible.